pkg/log/log_test.go

```diff
@@ -36,7 +34,7 @@ func (s *LogSuite) TestWithNilError(c *C) {
 func (s *LogSuite) TestWithNilContext(c *C) {
 	log.SetFormatter(&logrus.JSONFormatter{TimestampFormat: time.RFC3339Nano})
 	// Should not panic
-	WithContext(nil).Print("Message")
+	WithContext(context.TODO()).Print("Message")
```

Comment:
this is actually testing a nil context. please use https://github.com/golangci/golangci-lint#nolint